test/dynamo/test_unspec.py

```diff
@@ -659,7 +659,7 @@ def fn(x, y):
 
             cnt = CompileCounterWithBackend("aot_eager")
             fn_opt = torch._dynamo.optimize(cnt)(fn)
-            x = torch.tensor(9.734375, dtype=dtype, requires_grad=True)
+            x = torch.randn(5, dtype=dtype, requires_grad=True)
```

Comment:
I just changed this to be a non scalar tensor to make the test simpler, otherwise we would have different frame counts depending on if dtype == torch.float64. This is because were were assigning zf0 to 9.734375 and zf1 to the concrete value of y. So at the time of tensorify, we would tensorify away zf1 but not zf0 causing a restart.

I suppose this highlights one of the downsides of the current implementation where sometimes we will restart analysis when we don't really need to. I guess in the most ideal world at the time of tensorify we would know what was specialized before this point in time (eg. in arg_parser.h or in fake tensor prop). Then we could know which non tensorified away symfloats are okay.